### ISSUE TITLE
Suggestions to improve Galaxy HiCExplorer

### ISSUE THREAD

**Issue by gtrichard** [deeptools/HiCExplorer#226]
During my review of the wrappers to improve documentation of the Galaxy version, I noted a few suggestions that could be implemented in the future:

- Build a wrapper for hicQC in order to pass it multiple hicBuildMatrix log files without having to re-run hicBuildMatrix (the point is, when you generate new data, you can still add the log files in hicQC to compare them to previously processed data).

- hicCorrectMatrix wrapper: add the --perchr option to diagnostic_plot?

- hicInfo wrapper to add.

- hicFindTADs option to use z-score matrix as input in order to test multiple TAD calling parameters more quickly (it allows to not compute the z-score matrix everytime, which is the most demanding step in hicFindTADs in terms of memory and CPU usage).

- findRestSites wrapper. I have colleagues that want to start Hi-C on some non-model organism and having a small Galaxy wrapper to test different restriction enzymes on any genome could be cool (that way they don't need a bioinformatician / command-line user from the start to find the most suitable restriction enzyme before starting fine-tuning the wet-lab protocol).

**Comment by joachimwolff** [comment]
> Build a wrapper for hicQC in order to pass it multiple hicBuildMatrix log files without having to re-run hicBuildMatrix (the point is, when you generate new data, you can still add the log files in hicQC to compare them to previously processed data).
I think it is already covered by MultiQC.

Other suggestions: I agree on them.

**Comment by joachimwolff** [comment]
hicPlotTads: 
- add a selection to chose only region or chromosomeOrder

This would make it easier for beginners. They could select what they want to plot and would not get an error if they use both.

**Comment by joachimwolff** [comment]
Support for mcool files.

**Comment by cgirardot** [comment]
Hi
any more update on this? I would also enjoy wrappers for `findRestSites` and `hicInfo`. 
May I also suggest to expose the `--dpi` option in the `hicPlotMatrix` wrapper to increase png resolution (and possibly support PDF format).
Thanks

**Comment by joachimwolff** [comment]
https://github.com/galaxyproject/tools-iuc/pull/2590

Adds:
- All features from 3.2, 3.1, 3.0 and 2.2 releases
- dpi options for plots
- file format options for plots: svg, png or pdf
- using precomputed z-score for hicFindTads
- Adds findRestSite wrapper

Missing:
- mcool support
- hic file type support
